lib/commons/color/get-background-color.js

```diff
@@ -38,16 +46,45 @@ function getBgColor(elm, elmStyle) {
 	return bgColor;
 }
 
-function calculateObscuringAlpha(elmIndex, elmStack) {
+/**
+ * Determines overlap of node's content with a bgNode. Used for inline elements
+ * @private
+ * @param {Element} targetElement
+ * @param {Element} bgNode
+ * @return {Boolean}
+ */
+function contentOverlapping(targetElement, bgNode) {
+	// get content box of target element
+	// check to see if the current bgNode is overlapping
+	var targetRect = targetElement.getClientRects()[0];
+	var obscuringElements = document.elementsFromPoint(targetRect.left, targetRect.top);
+	if (obscuringElements) {
+		for(var i = 0; i < obscuringElements.length; i++) {
```

Comment:
I'd prefer we use array.find() instead, it's easier to read and harder to make bugs in (IMO). Not a big deal, just a suggestion for future updates.